### ISSUE TITLE
Adding more visual treats🎉

### ISSUE THREAD

**Issue by kishore007k** [Yog9/SnapScout#12]
## Customization🛠

<p>Here is the screenshot of the website with some tweakings</p>

I have added a new category `All` to the website

![Capture](https://user-images.githubusercontent.com/34863222/90343538-f31b6e00-e02e-11ea-9322-be28a8faa242.PNG)

**Comment by Yog9** [comment]
Hi @kishore007k I can see the screenshot but not your PR for this feature.

**Comment by kishore007k** [comment]
Yeah I juzt wanted to confirm about the issue so still I haven't raised a PR

**Comment by kishore007k** [comment]
@Yog9 Yeah I juzt wanted to confirm about the issue so still I haven't raised a PR
